TESTS/network/wifi/wifi_tests.h

```diff
@@ -0,0 +1,26 @@
+#ifndef WIFI_TESTS_H
```

Comment:
I believe every file in the codebase should have license header on the top, even test files.